### PR TITLE
BUG FIX urdb_parse TOU demand charge time steps

### PR DESCRIPTION
**Pull request by NLaws**
TOU demand charge time steps were zero indexed, which is incorrect for arrays that are used in reopt_model.jl (should be one indexed)

### CODE REVIEW

File: reo/src/urdb_parse.py [hunk 1]
```diff
@@ -639,7 +638,13 @@ def get_hours_in_month(self, month):
...
+        """
+        Get the TOU steps indexed on 1-8760 for hourly simulations
+        :param current_rate: RateData class instance
+        :param month: int
+        :param period: int
+        :return: list of ints
+        """
         step_array = []
         start_step = 1
         start_hour = 1
```
Comment (lixiangk1) [inline reo/src/urdb_parse.py:650]: delete also?

**Review by lixiangk1 — APPROVED** [review]

File: reo/src/urdb_parse.py (hunk 1 above)
Comment (NLaws) [reply reo/src/urdb_parse.py:650]: is it wrong? (I was trying to document how the function is meant to be used.)

Comment (lixiangk1) [reply reo/src/urdb_parse.py:650]: oh haha, I thought the comment was going to show up more line specific. just the start_hour=1 part

Comment (NLaws) [reply reo/src/urdb_parse.py]: gotchya: removed it in https://github.com/NREL/REopt_Lite_API/pull/228/commits/f9e2ba7d5606a6777b329125ef04ae7d9b51f82b

**Pull request merged**